### PR TITLE
lbxml2 - 2.9.5 - Update to latest version.  Use ICU package (that wor…

### PR DESCRIPTION
**Pull request by JPeterMugaas**
…ks better in the tests).  catalog-revert patch dropped because it's in the source-code.

libxslt - 1.3.0 - Update to latest version and rebuild with ICU-dependant libxml2

### CODE REVIEW

File: libxml2/PKGBUILD [hunk 1]
```diff
@@ -1,42 +1,68 @@
...
+apply_patch_p2_with_msg() {
+  for _patch in "$@"
+  do
+    msg2 "Applying $_patch"
+    patch -Nbp2 -i "${srcdir}/$_patch"
+  done
+}
+
+del_file_exists() {
+  for _fname in "$@"
```
Comment (jtanx) [inline libxml2/PKGBUILD]: What's wrong with calling `rm -rf ` or `rm -f` always?

Comment (JPeterMugaas) [reply libxml2/PKGBUILD]: Sometimes, that might trigger an error if the file or dir doesn't exist.

Comment (jtanx) [reply libxml2/PKGBUILD]: That only happens without the `-f` flag.

File: libxslt/libxslt-1.1.29-msys2.patch [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
+@@ -781,6 +781,9 @@
      i*:CYGWIN*:*)
  	echo ${UNAME_MACHINE}-pc-cygwin
  	exit ;;
 +    i*:MSYS*:*)
 +	echo ${UNAME_MACHINE}-pc-msys
 +	exit ;;
-     *:MINGW64*:*)
- 	echo ${UNAME_MACHINE}-pc-mingw64
+     *:MINGW*:*)
```
Comment (Alexpux) [inline libxslt/libxslt-1.1.29-msys2.patch]: This is wrong. Return as it was

**Pull request merged**